.github/ISSUE_TEMPLATE/deployment.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Deployment
+about: Use this template when you're going to deploy a new release of the extension and web app.
+title: Publish vX.X.X extension + web app
+labels: deployment
+---
+
+- [ ] Manual testing to confirm extension works with main flows
+  - [ ] Balances
+  - [ ] Send
+  - [ ] Swap
+  - [ ] Staking
+  - [ ] IBC
+  - [ ] Tx details
+  - [x] ~Do a full sync with with [assertRootValid()](https://github.com/penumbra-zone/web/blob/main/packages/query/src/block-processor.ts#L383-L395) enabled. Will throw if TCT roots are not correct.~ (Ignore for now.)
+- [ ] If there is already a changelist PR (like [this one](https://github.com/penumbra-zone/web/pull/799)), merge it. Otherwise, manually run `pnpm changeset version`.
+  - This step will publish updated npm packages.
+- [ ] Update [npm package version](https://github.com/penumbra-zone/web/blob/main/package.json#L3)
+- [ ] Update [manifest version](https://github.com/penumbra-zone/web/blob/main/apps/extension/public/manifest.json#L4) based on the extension's newly updated [`package.json` version](https://github.com/penumbra-zone/web/blob/main/apps/extension/package.json) in the extension.
+- [ ] [Create repo release](https://github.com/penumbra-zone/web/releases/new) with `vX.X.X` tag. Triggers approval to run chrome extension publishing.
+- [ ] Run `pnpm build` in the web repo's root. Then take minifront & node-status `dist` output and make PR against core repo to update node's frontends.
```

Comment:
After https://github.com/penumbra-zone/web/issues/857, we'll have a script to take care of the zipping